pkg/build/cmd/publishaws_test.go

```diff
@@ -168,13 +168,13 @@ type mockAwsMarketplaceDocker struct {
 	ImagePushError error
 }
 
-func (m *mockAwsMarketplaceDocker) ImagePull(ctx context.Context, refStr string, options types.ImagePullOptions) (io.ReadCloser, error) {
+func (m *mockAwsMarketplaceDocker) ImagePull(ctx context.Context, refStr string, options image.PullOptions) (io.ReadCloser, error) {
```

Comment:
Are we using the drone runner directly?  if so -- the version change will likely break:
https://github.com/drone-runners/drone-runner-docker/blob/v1.8.3/engine/engine.go#L234
😢